### PR TITLE
subnet: accelerate application controllers processing

### PR DESCRIPTION
**Pull request by Icarus9913**
1. modifty multiple controllers workers. Increase the application controllers workers to 5 to make it faster to create empty auto-created IPPool for SpiderSubnet feature. And set the subnet informer and IPPool informer workers to 3 is fine, because the auto-created IPPools Spec.IPs allocation from SpiderSubnet operation is serial.
2. change workqueue requeue delay duration from 1s to 5s, because there's no need to put the unknown processing error items back to workqueue immediately.
3. add spider subnet feature architecture picture

Signed-off-by: Icarus9913 <icaruswu66@qq.com>

**What this PR does / why we need it**:
improvement

**Which issue(s) this PR fixes**:
https://github.com/spidernet-io/spiderpool/issues/846

### CODE REVIEW

**Review by iiiceoo — APPROVED** [review]

File: cmd/spiderpool-controller/cmd/config.go [hunk 1]
```diff
@@ -82,8 +83,8 @@ var envInfo = []envConf{
 	{"VERSION", "", false, &controllerContext.Cfg.AppVersion, nil, nil},
 	{"SPIDERPOOL_ENABLE_SUBNET_DELETE_STALE_IPPOOL", "false", true, nil, &controllerContext.Cfg.EnableSubnetDeleteStaleIPPool, nil},
 	{"SPIDERPOOL_AUTO_IPPOOL_HANDLER_MAX_WORKQUEUE_LENGTH", "10000", true, nil, nil, &controllerContext.Cfg.IPPoolInformerMaxWorkQueueLength},
-	{"SPIDERPOOL_AUTO_IPPOOL_SCALE_RETRY_DELAY_DURATION", "1", true, nil, nil, &controllerContext.Cfg.WorkQueueRequeueDelayDuration},
-	{"SPIDERPOOL_IPPOOL_INFORMER_WORKERS", "5", true, nil, nil, &controllerContext.Cfg.IPPoolInformerWorkers},
+	{"SPIDERPOOL_WORKQUEUE_RETRY_DELAY_DURATION", "5", true, nil, nil, &controllerContext.Cfg.WorkQueueRequeueDelayDuration},
```
Comment (Icarus9913) [inline cmd/spiderpool-controller/cmd/config.go:86]: modify the re-queue delay duration form 1s to 5s

**Comment by weizhoublue** [comment]
pr 的 title 应该描述 本pr 优化了功能或者解决了什么问题， 在 release log 知道这个什么作用，
而不是 
modifty multiple controllers workers

**Comment by weizhoublue** [comment]
any update

**Comment by weizhoublue** [comment]
feature-new. pr ?

**Review by iiiceoo — APPROVED** [review]

**Pull request merged**